.github/workflows/UnitTests.yml

```
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.3', '1.6', 'nightly']
+        julia-version: ['1.3', '1.6', '1.7', 'nightly']
```

For what is worth, I usually test my packages on "oldest supported version", "latest stable version" and "nightly", that is
```suggestion
        julia-version: ['1.3', '1', 'nightly']
```
In this way I don't have to update the list of Julia versions every time there is a new release and hope that semver does the rest (if there are no breaking changes, what worked on old versions is supposed to keep working on newer versions)

---

Thanks for the comment, @giordano. We do not support anything below Julia 1.3 because of our dependency on binary builds (through Libsharp_jll), so your suggestion would lead to:

```
julia-version: ['1.3', 'nightly']
```

Is this correct?

---

I prefer to test both latest stable version and nightly because nightly sometimes break by itself. It's still useful to test nightly in case it introduces accidentally breaking changes, so that'd be good to report them as soon as possible